changelog.d/1016.doc.rst

```diff
@@ -0,0 +1 @@
+Render the unreleased ``attrs`` version change notes on the changelog page in docs.
```

Comment:
We don’t have a doc category, so this will have to do without a fragment.

Reply:
Oh, I didn't realize that. That's a power of habit :)

Did you consider including `CONTRIBUTING.md` into the Sphinx docs? It's pretty straightforward to add a single Markdown document there via MyST. No need to convert everything into the same format.

P.S. Feel free to try out https://github.com/apps/psf-chronographer if you want to verify the presense of the allowed change note types in PRs, like aiohttp does.

Reply:
Personally, I was a little lost when I couldn't locate this in the docs, among the top-level items. I think there was a link on one of the pages where I never land coming from Google.

Reply:
there's one on the github readme, the main docs page, and the pypi page :D

maybe i'll just add it to the list so it's less embedded